atm.js

```diff
@@ -1,3 +1,191 @@
+    $("#savings .balance").css("background", "rgb(232, 63, 63)");
+  } else {
+  savingsWithdrawFunction(withdrawAmount);
+  $(".input").val("");
+}
+});
+
+function savingsDepositFunction(depositAmount) {
+  var toNumber = parseFloat(depositAmount);
+  var toTwoDecimalPlaces = toNumber.toFixed(2);
+  var backToNumber = parseFloat(toTwoDecimalPlaces);
+  savingsTotal += backToNumber;
+  var finalNumber = "$" + parseFloat(savingsTotal.toFixed(2));
+  $("#savings .balance").html(finalNumber);
+}
+
+function savingsWithdrawFunction(withdrawAmount) {
+  var toNumber = parseFloat(withdrawAmount);
+  var toTwoDecimalPlaces = toNumber.toFixed(2);
+  var backToNumber = parseFloat(toTwoDecimalPlaces);
+  if (backToNumber > savingsTotal) {
+    $("#savings .balance").html("<p class='smallText'>Not Enough Funds</p>");
+    $("#savings .balance").css("background", "rgb(232, 63, 63)");
+    setTimeout(savingsReset, 2000);
+  } else if (backToNumber === savingsTotal) {
+    savingsTotal = 0;
+    $("#savings .balance").html("$0");
+    $("#savings .balance").css("background", "rgb(232, 63, 63)");
+  } else {
+    savingsTotal -= backToNumber;
+    var finalNumber = "$" + parseFloat(savingsTotal.toFixed(2));
+    $("#savings .balance").html(finalNumber);
+  }
+
+  //setTimeout function in order to re-display total value after withdraw has been rejected
+  function savingsReset() {
+    $("#savings .balance").html("$" + parseFloat(savingsTotal.toFixed(2)));
+    if (savingsTotal === 0) {
+      $("#savings .balance").css("background", "rgb(232, 63, 63)");
+    } else {
+      $("#savings .balance").css("background", "#E3E3E3");
+    }
+  };
+}
+
+
+
+
+
+
+
+/*********
+Pseudo Code
+**********/
+
+// DEPOSITS //
+
+//On click: Take value inputed by user, assign to deposit variable, and pass amount through as argument to deposit function
+
+//deposit function
+// parse deposit amount as a float to two decimal places
+// reparse amount since it will be a string
+// add dollar sign in front of value
+// append value as html to the total amount
+
+
+//On click: Take value inputed by user, assign to withdraw variable, and pass amount through as argument to withdraw function
+
+// WITHDRAWS //
+
+//withdraw function
+// parse withdraw amount as a float to two decimal places
+// reparse amount since it will be a string
+    //if variable stored value inputed by user is greater than total amount, do nothing or deny/ignore request
+    // display current total
+    //elseif variable stored value inputed by user is equal to total amount in account, total box turns red, display new total (which should be 0)
+    //if variable stored value inputed by user is less than total amount
+    // subtract stored variable value from the total amount, display new total
+        // add dollar sign in front of value
+        // append value as html to the total amount
+
+//ON CLICK Take value inputed by user and store to withdraw variable, parsed as a float to two decimal places
+
+
```

Comment:
Excellent pseudocode here! I can tell it really helped you out when you wrote your actual program.